pkg/errors/error.go

```diff
@@ -103,12 +103,12 @@ func GetStackTraceForError(err error) errors.StackTrace {
 }
 
 // NewValidation creates a validation error
-func NewValidation(group string, err error) *StatusError {
+func NewValidation(msg string, err error) *StatusError {
```

Comment:
Was just confused by the `group` param name

Reply:
use the group in other errors.  The idea is that it should be something like "account" or "lease" so we can create a similar structure in the error messages across the different resources.

Reply:
oh I see, like it would say "account validation error: ....."?

Reply:
I'll revert this change